chat/src/components/chat-provider.tsx

```diff
@@ -57,7 +57,8 @@ export function ChatProvider({ children }: PropsWithChildren) {
   const [serverStatus, setServerStatus] = useState<ServerStatus>("unknown");
   const eventSourceRef = useRef<EventSource | null>(null);
   const searchParams = useSearchParams();
-  const agentAPIUrl = searchParams.get("url") || window.location.origin;
+  const defaultAgentAPIURL = new URL("../../", window.location.href).toString();
```

Comment:
Why the default Agent API URL has `../..`?

Reply:
I would left a comment to explain that in code. I see myself forgetting about this very easily.

Reply:
Done.